versions/0.0.2/README.md

```diff
@@ -0,0 +1,13 @@
+# API Specification
+This directory contains the various parts of the open API specification for exchanging data and operational control in indoor agricultural environments.  
+
+The specification is divided into the following sub-sections to make it easier to follow...
+
+* **General** - contains all of the essential rules that need to be followed, including common concepts, common data structures, timestamp formats, error handling, and so on.
```

Comment:
I think it could be helpful to make these bold headings into links so new readers can click into each sub-section. I believe Github supports relative links. For example the general heading would become ```**[General](versions/0.0.2/general.md)**```